.eslintrc.yaml

```diff
@@ -4,7 +4,6 @@ parserOptions:
   ecmaFeatures:
     jsx: true
     modules: true
-  project: './tsconfig.json'
```

Comment:
As far as I know, this is the default, there is no need to define it.
Of course I can revert it if you want to keep it.